src/seedu/addressbook/commands/Command.java

```diff
@@ -70,4 +70,8 @@ public int getTargetIndex() {
     public void setTargetIndex(int targetIndex) {
         this.targetIndex = targetIndex;
     }
+    
+    public boolean isMutating(){ 
+        return false; 
```

Comment:
Here you specified the default return value as false, which helps to reduce the amount of code. However, what are the implications on understandability and intuitiveness of the code when an arbitrary default value is chosen? Maybe you can consider making this method `abstract` to force all child classes to declare whether it is a mutating command or not rather than depend on an arbitrary decision.